installer/targets/robot/install.bash

```diff
@@ -1,2 +1,9 @@
+### Make sure remote launching works ###
 tue-install-cp ros-bash-and-run.sh ~/.ros-bash-and-run.sh
 chmod +x ~/.ros-bash-and-run.sh
+
+### Allow user to access serial interfaces ###
```

Comment:
Which serial interfaces?

Reply:
For example the hokuyo or (in the future) the dynamixels